### PR TITLE
Detect existing RFCs

### LINKED ISSUE #15: Somehow link RFCs if they're created by re-running deployments

### PR DESCRIPTION
**Pull request by GaryGSC**
Closes #15. Hopefully. You can read all about my thought process over there.

I haven't actually seen what happens when we try ending a standard change for the second time with [`github-action-end-standard-change`](https://github.com/byu-oit/github-action-end-standard-change), so we shouldn't release this to `v1` until we're comfortable with that interaction.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -35,19 +35,35 @@ async function run () {
...
+    })
+    if (alreadyCreatedRfc) {
+      warning('An existing RFC was found!')
+      console.log(`RFC Number: ${alreadyCreatedRfc.number}
+Link to RFC: https://${servicenowHost}/change_request.do?sysparm_query=number=${alreadyCreatedRfc.number}
+Created on: ${alreadyCreatedRfc.sys_created_on}
+Last updated on: ${alreadyCreatedRfc.sys_updated_on}`)
+      // Set outputs for GitHub Actions
+      setOutput('change-sys-id', alreadyCreatedRfc.sys_id)
+      setOutput('work-start', convertServicenowTimestampFromMountainToUtc(alreadyCreatedRfc.work_start))
```
Comment (GaryGSC) [inline index.js]: I'm having a hard time telling if we need `convertServicenowTimestampFromMountainToUtc()` in this case, due to some existing bad data (#29, which was recently fixed). I actually suspect that we _don't_ need it here.

**Comment by GaryGSC** [comment]
Can I get some eyes on this so that we can merge it into `staging` and kick the tires a little?

**Comment by GaryGSC** [comment]
@blakemorgan, @stuft2, this should hopefully make you two happy. 😄

**Comment by GaryGSC** [comment]
Eh. I guess I can try it from here (`uses: byu-oit/github-action-start-standard-change@detect-existing-rfcs`)

File: index.js (hunk 1 above)
Comment (stuft2) [reply index.js]: I'm not familiar enough on the reasoning for this function to give a recommendation

**Review by stuft2 — DISMISSED** [review]
I'm interested in seeing how this behaves. I'm unfamiliar with the descriptionLike query param but the code looks clean and correct to me.

**Comment by GaryGSC** [comment]
I found [some docs for `sysparm_query`](https://developer.servicenow.com/dev.do#!/reference/api/orlando/rest/c_TableAPI). So, that beats divining it from the URL in ServiceNow, as I did originally. Unfortunately, I can't deep-link to the spot where it talks about "LIKE", so you'll need to <kbd>CTRL</kbd> + <kbd>F</kbd> for it.

**Review by stuft2 — APPROVED** [review]

**Pull request merged**